crates/sui-types/src/messages_checkpoint.rs

```diff
@@ -455,6 +462,17 @@ impl CheckpointContents {
         self.transactions.iter()
     }
 
+    pub fn enumerate_transactions(
+        &self,
+        ckpt: &CheckpointSummary,
+    ) -> impl Iterator<Item = (u64, &ExecutionDigests)> {
```

Comment:
Please document the semantics of the returned tuple

Reply:
done